### ISSUE TITLE
Queue syncing? | Test case 11

### ISSUE THREAD

**Issue by niclee1219** [nus-cs2030/2223-s2#797]
# Summary

send help 

## Description:

honestly not sure why 15 is served last.. 

## Screenshots

Screenshots (if any)
<img width="716" alt="Screenshot 2023-04-07 at 11 14 21 PM" src="https://user-images.githubusercontent.com/20497341/230632876-76a2ef5c-1a34-4945-9cf3-1ed9a7b95913.png">

**Comment by calvnleong** [comment]
Could it be that the start time of the wait/serve event for 15 is after the wait/serve event of 16 and 17 when you created the event

**Comment by cheng-yee** [comment]
Your event time in line 67 and 68 is also before event time in 65 and 66 (not chronologically correct). Maybe check if your string output values for your events are in sync with the event time and customer number values of events compared by your priority queue when it polls an event. Hope this helps nicholas!!

**Comment by easypzlemonsqueezy** [comment]
yea as above i suspect that it's either your event comparator or your string output that is messing your output sequence! you might wanna double check your code and it might possibly be due to an extra/missing +1 somewhere

**Comment by niclee1219** [comment]
[FIXED]

please refer to this thread if you need further assistance. thanks all!
https://github.com/nus-cs2030/2223-s2/issues/745

TL;DR [remove customer at buffer wait instead of serve]
